docs/stream.md

```diff
@@ -114,6 +116,39 @@ Argument     | Type                 | Required | Description
 
 ---
 
+##### Stream.scan
+
+Creates a new stream with the results of calling the function on every value in the stream with an accumulator and the incoming value.
+
+`stream = Stream.scan(fn, accumulator, stream)`
+
```

Comment:
Should this be `var` `let` or `const`?

I think it would be nice to show a usage example too e.g.

```js
const numbers = Stream()
const sum = Stream.scan( (a,b) => a+b, 0, numbers )

numbers(1)
sum() //=> 1
numbers(2)
sum() //=> 3
number(3)
sum() //=> 6
```

Might be helpful to refer to `accumulator` as `initial`, even though accumulator is technically accurate, I think `initial` will help people new to streams understand how to get started with scan.

Could also be good to point out that `scan` is useful for implementing patterns similar to Redux or the Elm Architecture in a light weight way:

```js
const actions = Stream()
const state = Stream.scan( update, { count: 0 }, actions )

function update(state, action){
  return (
    action.type == 'INCREMENT'
      ? Object.assign(state, { count: state.count+ 1 })
    : action.type == 'DECREMENT'
      ? Object.assign(state, { count: state.count - 1 })
    : state
  )
}

state() //=> { count: 0 }
actions({ type: 'INCREMENT' })
state() //=> { count: 1 }
actions({ type: 'INCREMENT' })
state() //=> { count: 2 }
```